src/ios/CDVFileTransfer.m

```diff
@@ -552,6 +552,8 @@ - (void)connectionDidFinishLoading:(NSURLConnection*)connection
 
     if (self.direction == CDV_TRANSFER_UPLOAD) {
         uploadResponse = [[NSString alloc] initWithData:self.responseData encoding:NSUTF8StringEncoding];
+        if (uploadResponse == nil)
+            uploadResponse = [[NSString alloc] initWithData: self.responseData encoding:NSISOLatin1StringEncoding];
```

Comment:
Did you try encoding Windows-1252 or CP-1252 on windows? Sorry, I don't have such environment.